chapters/samplers.txt

```diff
@@ -856,3 +863,26 @@ include::{generated}/validity/protos/vkDestroySamplerYcbcrConversion.txt[]
 --
 
 endif::VK_VERSION_1_1,VK_KHR_sampler_ycbcr_conversion[]
+
+ifdef::VK_EXT_custom_border_color[]
+
+[open,refpage='VkSamplerCustomBorderColorCreateInfoEXT',desc='Structure specifying custom border color',type='structs']
+--
+
+Applications may wish to provide a custom border color which is not listed under
```

Comment:
I'd avoid the non-normative use of "may" and not personify applications (apps don't "wish" to do things.  Developers do.  Apps just do things).

Suggest something like:

"Applications may: provide a border color not listed under ename:VkBorderColor by adding the sname:VkSamplerCustomBorderColorCreateInfoEXT to the pname:pNext chain of the slink:VkSamplerCreateInfo structure."

Reply:
s/may/can